### ISSUE TITLE
General: Other sources are not accessible for keyboard user 

### ISSUE THREAD

**Issue by leomendoza123** [ORCID/orcid-angular#1086]
**Describe the bug**
Other sources are not accessible 
**To Reproduce**
Try to open "other sources" via keyboard
**Expected behavior**
The app should be usable via keyboard

**Screenshots**
![image](https://user-images.githubusercontent.com/2119626/132801448-668351f2-1480-4556-a8f1-3a909d4794ec.png)
